options/m_option.c

```diff
@@ -1438,6 +1438,21 @@ static char **separate_input_param(const m_option_t *opt, bstr param,
         str = bstr_cut(str, 1);
         n++;
     }
+    // Backwards compatibility for path list options.
```

Comment:
This will not work in practice. There is no sane way to provide backward compatibility.

If you had a list separated with `;` you likely didn't have `,` escaped, so it will already break the list in the main loop above.

By doing this we basically force to escape both separators.

Reply:
I know. It's mentioned in the commit message. There is breakage no matter what, but I figured it was worth at least trying if the user doesn't have `,` in there.

Reply:
It's really annoying, change like that will break everything. And even if compatibility is provided, all users are expected to migrate. ¯⁠\\⁠_⁠(⁠ツ⁠)⁠_⁠/⁠¯

Reply:
Are there really that many people that use paths with `,` in them? I know the number isn't zero, but I feel it shouldn't be *too* many. We can always wait a couple of stable releases or so before yanking it.

Reply:
I mean in general switch from `:`/`;` to `,`. Scripts/configs and so on... Though I like the change, because I always need to check what is the separator for option X and system Y..

Reply:
There might be some pain but I think it's for the greater good.